src/services/ui/src/utils/zod.ts

```diff
@@ -98,6 +98,10 @@ export const zAmendmentOriginalWaiverNumberSchema = z
     message:
       "According to our records, this 1915(b) Waiver Number does not yet exist. Please check the 1915(b) Initial or Renewal Waiver Number and try entering it again.",
   })
+  .refine(async (value) => canBeRenewedOrAmended(value), {
```

Comment:
@mdial89f This is what I mean -- that `value` is somehow the `id` attribute and I'm not sure where that's all wired up -- that'd be the tricky bit.